Source/Scene/Cesium3DTileset.js

```diff
@@ -0,0 +1,1821 @@
+            },
+            set : function(value) {
+                this._modelMatrix = Matrix4.clone(value, this._modelMatrix);
+                if (defined(this._root)) {
+                    // Update the root transform right away instead of waiting for the next update loop.
+                    // Useful, for example, when setting the modelMatrix and then having the camera view the tileset.
+                    this._root.updateTransform(this._modelMatrix);
+                }
+            }
+        },
+
+        /**
+         * Returns the time, in milliseconds, since the tileset was loaded and first updated.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {Number}
+         * @readonly
+         */
+        timeSinceLoad : {
+            get : function() {
+                return this._timeSinceLoad;
+            }
+        },
+
+        /**
+         * The total amount of GPU memory in bytes used by the tileset. This value is estimated from
+         * geometry, texture, and batch table textures of loaded tiles. For point clouds, this value also
+         * includes per-point metadata.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {Number}
+         * @readonly
+         *
+         * @see Cesium3DTileset#maximumMemoryUsage
+         */
+        totalMemoryUsageInBytes : {
+            get : function() {
+                var statistics = this._statistics;
+                return statistics.texturesByteLength + statistics.geometryByteLength + statistics.batchTableByteLength;
+            }
+        },
+
+        /**
+         * @private
+         */
+        styleEngine : {
+            get : function() {
+                return this._styleEngine;
+            }
+        },
+
+        /**
+         * @private
+         */
+        statistics : {
+            get : function() {
+                return this._statistics;
+            }
+        }
+    });
+
+    /**
+     * Marks the tileset's {@link Cesium3DTileset#style} as dirty, which forces all
+     * features to re-evaluate the style in the next frame each is visible.
+     */
+    Cesium3DTileset.prototype.makeStyleDirty = function() {
+        this._styleEngine.makeDirty();
+    };
+
+    /**
+     * Loads the main tileset.json or a tileset.json referenced from a tile.
+     *
+     * @private
+     */
+    Cesium3DTileset.prototype.loadTileset = function(tilesetUrl, tilesetJson, parentTile) {
+        var asset = tilesetJson.asset;
+        //>>includeStart('debug', pragmas.debug);
+        Check.typeOf.object('tilesetJson.asset', asset);
+        if (asset.version !== '0.0' && asset.version !== '1.0') {
+            throw new DeveloperError('The tileset must be 3D Tiles version 0.0 or 1.0.  See https://github.com/AnalyticalGraphicsInc/3d-tiles#spec-status');
+        }
+        //>>includeEnd('debug');
+
+        var statistics = this._statistics;
+
+        // Append the version to the basePath
```

Comment:
This should probably say `tileset version` to clarify that it's not the spec version.